05_classwork_strange_button/index_strange_button.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<!--[if lte IE 6]>
+<html class="preIE7 preIE8 preIE9">
+   <![endif]-->
+<!--[if IE 7]>
+   <html class="preIE8 preIE9">
+      <![endif]-->
+<!--[if IE 8]>
+      <html class="preIE9">
+         <![endif]-->
+<!--[if gte IE 9]><!-->
+<html>
+<!--<![endif]-->
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <title>My ReDi Project</title>
+    <link rel="stylesheet" href="strange_button.css" type="text/css">
+</head>
+
+<body>
+    <button class="strangeButton" id="strangeButton">strange button</button>
+    <script>
+        let strangeButtonEL = document.getElementById("strangeButton");
+        let newButtonEL = document.createElement("button");
+        let remButtonEl = document.createElement("button")
```

Comment:
Do not omit ; at the end of the statements. It's working, but it's better to have them.
Check code below for other missing ;

Reply:
Checked, fixed.